src/utils/misc.js

```diff
@@ -43,19 +43,21 @@ const runCommand = (command, args, options) => {
     let stdout = '';
     if (result.stdout) {
       result.stdout.on('data', data => {
-        stdout += data.toString();
+        const str = data.toString();
+        stdout += str;
         if (global.argOpts.debug) {
-          console.log(colors.green(stdout));
+          process.stdout.write(colors.green(str));
```

Comment:
Why the change from `console.log` to `process.stdout`?

Reply:
`console.log` appends a newline every call, but the stdout from the external command already has the newlines.